### ISSUE TITLE
Training set and test set partitioning

### ISSUE THREAD

**Issue by shuangzixing** [detectRecog/CCPD#52]
Hey, could please describe the implementation details,  such as the training set and test set partitioning and training process of other subset(excluding the CCPD-Base). These question are not described in detail in the paper.

**Comment by jfzhuang** [comment]
I also have the same question. Please provide more necessary details to aid reproduction.

**Comment by Harsha-1729** [comment]
> I also have the same question. Please provide more necessary details to aid reproduction.

Hello I also have the same question. Were you able to solve it?

**Comment by Harsha-1729** [comment]
> Hey, could please describe the implementation details, such as the training set and test set partitioning and training process of other subset(excluding the CCPD-Base). These question are not described in detail in the paper.

Hello I also have the same question. Were you able to solve it?
